packages/blade/rollup.config.mjs

```diff
@@ -186,21 +240,35 @@ const config = () => {
       .map((exportCategory) => [getCSSVariablesConfig({ exportCategory })])
       .flat();
   }
+
+  const components = glob.sync(`src/components/**/index.ts`);
+  const tokens = glob.sync(`src/tokens/index.ts`);
+  const utils = glob.sync(`src/utils/index.ts`);
```

Comment:
not really a glob right? should we just use paths directly?

Reply:
right.